### DISCUSSION TITLE
Allow ?? return

### DISCUSSION THREAD

**Discussion by ahdung** [dotnet/csharplang#4820] [Language Ideas]
`x ?? throw ex` is great, I hope also can do `x ?? return` to instead of:
```c#
var result = await FooAsync().ConfigureAwait(false);
if (result == null)
  return; // jump out of method
  // or return something if method with return parameter.

Use(result);
```

**Comment by Joe4evr** [comment]
#867

**Comment by huoyaoyuan** [comment]
This was rejected because the team want to extend the meaning of `return` in the future, thus don't want a feature adds limitation.
